tests/assert_eq.rs

```diff
@@ -21,15 +30,24 @@ fn assert_eq() {
     let x = Some(Foo { lorem: "Hello World!", ipsum: 42, dolor: Ok("hey".to_string())});
     let y = Some(Foo { lorem: "Hello Wrold!", ipsum: 42, dolor: Ok("hey ho!".to_string())});
 
-    assert_eq!(x, y, "custom panic message");
+    assert_eq!(x, y);
```

Comment:
I assumed this test were originally meant for testing without the custom message. Is it OK? :)